src/test/resources/config/application-artemis.yml

```diff
@@ -18,19 +18,16 @@ artemis:
             user-dn: <user-dn>
             password: <password>
             base: <base>
-        internal-admin:
-            username: artemis_admin
-            password: artemis_admin
     version-control:
-        url: https://bitbucket.ase.in.tum.de
         user: <username>    # e.g. ga12abc
         password: <password>
-        secret: <token>                 # VCS API token giving Artemis full Admin access. Not needed for Bamboo+Bitbucket
-        ci-token: <token from the CI>   # Token generated by the CI (e.g. Jenkins) for webhooks from the VCS to the CI. Not needed for Bamboo+Bitbucket
+        url: https://bitbucket.ase.in.tum.de
+        secret: -2HXoj9GZVxzzFQozxsG                 # VCS API token giving Artemis full Admin access. Not needed for Bamboo+Bitbucket
+        ci-token: d2fc9e8ecf37baa22e0a2b6382c88a10   # Token generated by the CI (e.g. Jenkins) for webhooks from the VCS to the CI. Not needed for Bamboo+Bitbucket
```

Comment:
That should probably not in here :sweat_smile:

Reply:
We are using those values when we generate the Jenkins buildplan configurations for the exercises, and it turns out that the special characters like <> were throwing parsing exceptions. I replaced the actual tokens with placeholder values without the special characters now.